functional_tests/internal/chart.go

```diff
@@ -46,6 +46,7 @@ func ChartInstallOrUpgrade(t *testing.T, testKubeConfig string, valuesFile strin
 	install := action.NewInstall(actionConfig)
 	install.Namespace = Namespace
 	install.ReleaseName = chartReleaseName
+	install.Wait = true
```

Comment:
just thinking loudly ;) 
do we want to set also `Timeout` here? in case of failures with installation 
`install.Timeout = time.Duration(120) * time.Second`

Reply:
`Timeout` is already set on the next line :)

Reply:
ohh, did not notice that 
👍